hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/ClassInfo.kt

```diff
@@ -43,7 +45,8 @@ internal fun ClassInfo(classNode: ClassNode): ClassInfo? {
         FieldId(classNode, fieldNode) to FieldInfo(
             fieldId = FieldId(classNode, fieldNode),
             isStatic = fieldNode.access and (Opcodes.ACC_STATIC) != 0,
-            initialValue = fieldNode.value
+            initialValue = fieldNode.value,
+            changeIndicatorHash = getResourceContentHash(fieldNode)
```

Comment:
Initial thought here:
When I read this, I might think 'oh if this hash is changed, then the field is dirty', but it is more like an 'additional' hash: It might still be dirty, even if this 'additional' hash is unchanged (e.g. by the initial value changing): 

Do you think we can help with intuition by calling it 'additionalHash', or 'extraHash' + some documentation?

Reply:
How about additionalChangeIndicatorHash + doc?

When I see `additionalHash`, I have a question "hash of what?"